### ISSUE TITLE
Why did my learning rate drop from the initial lr

### ISSUE THREAD

**Issue by sjchasel** [Tony-Y/pytorch_warmup#12]
In every batch, I execute 
```python
loss.backward()
optimizer.zero_grad()
optimizer.step()
with warmup_scheduler.dampening():
    lr_scheduler.step()
```

It doesn't have a warm up process.

**Comment by Tony-Y** [comment]
Does the example code work in your environment?
https://github.com/Tony-Y/pytorch_warmup/blob/master/examples/emnist/main.py

**Comment by Tony-Y** [comment]
Your code does not optimize model parameters at all because `optimizer.zero_grad()` is called after `loss.backward()`. If you cannot understand this, please read the following tutorial:

https://pytorch.org/tutorials/beginner/basics/optimization_tutorial.html#optimizer

**Comment by Tony-Y** [comment]
Did you solve this issue?
